### ISSUE TITLE
limit_seq(tan(n)) and limit_seq(sin(n)/cos(n))

### ISSUE THREAD

**Issue by ghost** [sympy/sympy#14245]
```
>>> limit_seq(tan(n))
<-oo, oo>
>>> limit_seq(sin(n)/cos(n))
    ⎛sin(n)⎞
lim ⎜──────⎟
n─→∞⎝cos(n)⎠
```

**Comment by Roland-Sands** [comment]
> the return value of <-oo, oo> says "any subsequential limits are between -oo and oo" which is about as good as saying nothing

I agree, although I would go further
I think `<-a, b>` should only be returned when the user is explicitly asking for a heuristic

**Comment by Roland-Sands** [comment]
> I think <-a, b> should only be returned when the user is explicitly asking for a heuristic

Or better yet, in my opinion, create a new function, `infimums`
`infimums(a_n)` would return `(b, c)` where `b` is the limit infimum of `a_n` and `c` is the limit supremum of 
`a_n`, with it being made clear that these evaluations are heuristic

**Comment by leosartaj** [comment]
Hi,

IMHO,

I don't really like ``<-oo, oo>``, as it's too ambiguous for me. Did the algorithm fail? Is the limit not defined? It's hard to know. I really like the readability of something like 

```
    ⎛sin(n)⎞
lim ⎜──────⎟
n─→∞⎝cos(n)⎠
```

It's what I am use to in SymPy (returning unevaluated results when algorithms are not able to handle cases).

**Comment by anutosh491** [comment]
The following results above are responsible for the differing answers , should we expect an answer in `AccumBounds` for `(tan(1/n)).as_leading_term(n)` instead of `self`  ?? Will give it a thought

```
>>> (sin(1/n)).as_leading_term(n)
AccumBounds(-1, 1)

>>> (1/cos(1/n)).as_leading_term(n)
AccumBounds(-oo, oo)

>>> (tan(1/n)).as_leading_term(n)
tan(1/n)

>>> (sin(1/n)/cos(1/n)).as_leading_term(n)
AccumBounds(-oo, oo)
```

**Comment by anutosh491** [comment]
I just realized this has been fixed by one of my pr's and both give consistent results now . Could be marked as easy to fix for adding tests or could be closed !
```
>>> limit_seq(tan(n))
AccumBounds(-oo, oo)
>>> limit_seq(sin(n)/cos(n))
AccumBounds(-oo, oo)
```

**Comment by anutosh491** [comment]
This issue has been fixed by #22905 . And much more complex `limits` at `oo`(shown below) were tested .....
```
    assert limit(sin(x)**2 - cos(x) + tan(x)*csc(x), x, oo) == AccumBounds(S.NegativeInfinity, S.Infinity)
    assert limit(sin(x)**2 - cos(x) + sin(x)*cot(x), x, oo) == AccumBounds(S.NegativeInfinity, S.Infinity)
    assert limit(cot(x) - tan(x)**2, x, oo) == AccumBounds(S.NegativeInfinity, S.Infinity)
    assert limit(cos(x) - tan(x)**2, x, oo) == AccumBounds(S.NegativeInfinity, S.One)
    assert limit(sin(x) - tan(x)**2, x, oo) == AccumBounds(S.NegativeInfinity, S.One)
    assert limit(cos(x)**2 - tan(x)**2, x, oo) == AccumBounds(S.NegativeInfinity, S.One)
    assert limit(tan(x)**2 + sin(x)**2 - cos(x), x, oo) == AccumBounds(-S.One, S.Infinity) 
```
Hence not sure if implementing a test function for this issue adds any more value to the Pr overall , so closing !
